sympy/vector/deloperator.py

```diff
@@ -49,7 +49,8 @@ def gradient(self, scalar_field, doit=False):
         >>> from sympy.vector import CoordSysCartesian
         >>> C = CoordSysCartesian('C')
         >>> C.delop.gradient(9)
-        (Derivative(9, C.x))*C.i + (Derivative(9, C.y))*C.j + (Derivative(9, C.z))*C.k
+        (Derivative(9, C.x))*C.i + (Derivative(9, C.y))*C.j +
+            (Derivative(9, C.z))*C.k
```

Comment:
Running the `bin/doctest sympy/vector/deloperator.py` does not reveal any problems, all tests pass.